### PR TITLE
dotCMS/core#24971 [UI] Minimum length of experiment should be 14 days and max 90 days

### LINKED ISSUE #24971: [UI] Minimum length of experiment should be 14 days and max 90 days
### Parent Issue
#24343

### User Story
As an Experiments User, I want to be able to have my Experiments running at least for two weeks and max 90 days
Right now no minimum is required, which means, the User can set up an Experiment to last one day.

### Acceptance Criteria

- Experiments having a minimum of 14 days of duration.
- Trying to select less that this period from the UI should not be possible
- The max duration of an experiment is 90 days. 

- Related to #24941

### dotCMS Version
master

### Proposed Objective
_to_define_

### Proposed Priority
_to_define_

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.
_to_define_

### Assumptions & Initiation Needs
_to_define_

### Quality Assurance Notes & Workarounds
_to_define_

### Sub-Tasks & Estimates

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes

- Experiments having a minimum of 14 days of duration.
- The max duration of an experiment is 90 days.
- Trying to select less that this period from the UI should not be possible

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.spec.ts [hunk 1]
```diff
@@ -151,6 +152,24 @@ describe('DotExperimentsConfigurationSchedulingAddComponent', () => {
         jasmine.clock().uninstall();
     });
 
+    it('max end date date should be 90 days', function () {
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.spec.ts:155]: arrow function

**Review by fmontes — APPROVED** [review]

**Review by alfredo-dotcms — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts [hunk 2]
```diff
@@ -49,6 +49,10 @@ export class DotExperimentsConfigurationSchedulingAddComponent implements OnInit
     today = new Date();
     initialDate = new Date();
     minEndDate: Date;
+    maxEndDate: Date;
+
+    private time14days = 12096e5;
```
Comment (oidacra) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts:54]: Add this to the constant file of dotExperiments.

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.spec.ts (hunk 1 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.spec.ts]: done

File: core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts]: done

**Review by rjvelazco — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Pull request merged**